report_py3o/models/py3o_report.py

```diff
@@ -0,0 +1,370 @@
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import base64
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+from base64 import b64decode
+from contextlib import closing
+from io import BytesIO
+from zipfile import ZIP_DEFLATED, ZipFile
+
+import pkg_resources
+
+from odoo import _, api, fields, models, tools
+
+from ._py3o_parser_context import Py3oParserContext
+
+logger = logging.getLogger(__name__)
+
+try:
+    from py3o.template import Template
```

Comment:
```suggestion
    # workaround for https://github.com/edgewall/genshi/issues/15
    # that makes runbot build red because of the DeprecationWarning
    with warnings.catch_warnings():
        warnings.filterwarnings("ignore", category=DeprecationWarning)
        from py3o.template import Template
```